scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -129,8 +129,11 @@ class Router(formatOps: FormatOps) {
           Split(NoSplit, 0)
         )
       // Import
-      case FormatToken(_: T.Dot, _: T.LeftBrace | _: T.Underscore, _)
-          if existsParentOfType[Import](rightOwner) =>
+      case FormatToken(
```

Comment:
should this just be `case FormatToken(_: T.Dot, _, _)`?

Reply:
good point, it works as well!